frontend/src/components/common/Button/TButton.vue

```diff
@@ -5,42 +5,23 @@ Use of this software is governed by the Business Source License
 included in the LICENSE file.
 -->
 <script setup lang="ts">
-import type { StyleValue } from 'vue'
-import { computed, toRefs } from 'vue'
+import type { ButtonHTMLAttributes } from 'vue'
 
 import type { IconType } from '@/components/common/Icon/TIcon.vue'
 import TIcon from '@/components/common/Icon/TIcon.vue'
 
-type Props = {
+interface Props extends /* @vue-ignore */ Omit<ButtonHTMLAttributes, 'type'> {
   type?: 'primary' | 'secondary' | 'subtle' | 'subtle-xs' | 'compact' | 'highlighted'
-  toggle?: boolean
-  disabled?: boolean
   icon?: IconType
-  danger?: boolean
```

Comment:
did `danger` go away?

Reply:
It was not in use for this particular component so i dropped it, KISS and all that

Trivial to return if required